### PR TITLE
DDP-6043: Add validation to block entering future dates for DOD

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
Only item in ticket that seems actionable. Kicking it back to David O for further feedback.

### CODE REVIEW

File: study-builder/studies/rarex/i18n/en.json [hunk 1]
```diff
@@ -634,7 +634,7 @@
...
+  "ds_general_research": "<b>General Research</b><br/> This is the broadest type of research. When you choose General Research researchers may use your data for:<br/><p class=\"secondary-text\">a. <b>Health/Medical/Biomedical Research</b><br>Researchers can access and use your data to learn more about a health condition, its causes, symptoms, progression and treatments. This type of research could include research on any health condition, even if it is not a rare disease.<br/><br/>and</p><br/><br/><p class=\"secondary-text\">b. <b>Other kinds</b> of studies that are not related to health such as <ul class=\"content-list\"><li>Research on age, race, and ethnicity</li><li>Research studying traits such as how long people live or how easily they may get sick</li><li>Research about genetic traits of different populations</li><li>Studies to develop survey questions to improve research</li></ul>or",
```
Comment (MocanaAtBroad) [inline study-builder/studies/rarex/i18n/en.json:637]: This was out of scope, but just corrected a JSON syntax error in file. Escaping double-quotes.

Comment (yufengwng) [reply study-builder/studies/rarex/i18n/en.json:637]: Saw this as well and was planning to fix it. Thanks for putting this in.

File: study-builder/studies/rarex/legacy.conf [hunk 2]
```diff
@@ -191,7 +191,29 @@
...
-            "validations": []
+            "validations": [
+              {
+                "ruleType": "DATE_RANGE",
+                "startDate": "1898-01-01",
+                "endDate": null,
+                "useTodayAsEnd": true,
+                "hintTemplate": {
+                  "templateType": "TEXT",
+                  "templateText": "$dod_hint",
```
Comment (yufengwng) [inline study-builder/studies/rarex/legacy.conf:202]: I think you meant `dob`, but either way is fine.

**Review by yufengwng — APPROVED** [review]

File: study-builder/studies/rarex/legacy.conf (hunk 2 above)
Comment (MocanaAtBroad) [reply study-builder/studies/rarex/legacy.conf:202]: No. I meant dod. They erroneously call it DATE_OF_BIRTH. Did not want to compound error.

Comment (yufengwng) [reply study-builder/studies/rarex/legacy.conf:202]: Ah got it. 👍

**Pull request merged**